shaders/DefaultParticleShader.geom

```diff
@@ -6,7 +6,8 @@ layout (triangle_strip, max_vertices = 4) out;
 out vec2 UV;
```

Comment:
There are a lot of tabs and uneven linear spacing in this file. I don't know if it matters since this is a geometry shader but if it does matter then it should be fixed.